### ISSUE TITLE
Add instructions on how to feed back or query the rules

### ISSUE THREAD

**Issue by Melissa37** [JATS4R/jats-validator-ui#3]
Hi @hubgit 

Someone emailed some steering committee members with feedback about the Schematron rules and this led to a request to add something to the validator web page to indicate how people should feedback.

Could we add a line:
If you have any queries, comments or feedback about the validation rules please follow this link [https://github.com/JATS4R/jats-schematrons/issues] to open a ticket. Alternatively please feel free to email info@jats4r.org 

Not sure whether you wanted to add something about feedback/contributions to the code? I wanted to avoid using the term github when feeding back about the rules to not put off non-technical people!

Thanks

**Comment by hubgit** [comment]
Good idea - shall we put the bulk of that in the https://github.com/JATS4R/jats-schematrons README and link there from the validator client?

**Comment by Melissa37** [comment]
Hummm, might be offputting? Unless we added screenshots of how to add an issue?!

Just super aware of how github and terms like `read-me` can scare some production folk :-)
